src/KM_Defaults.pas

```diff
@@ -674,6 +674,11 @@ interface
   icRed    = $FF0707FF;
 
   icSteelBlue = $FFA56D53; // Selection color
+  icRoyalYellow = $FF4AC7FF;
```

Comment:
Please provide comments as to what these CONSTANT colors mean and where they are used and what for.